src/lib/services/web3-onboard.ts

```diff
@@ -92,9 +92,9 @@ export async function getOnboard() {
 					version: '1.0.0',
 					termsUrl: 'https://www.blocknative.com/terms-conditions',
 					privacyUrl: 'https://www.blocknative.com/privacy-policy'
-				},
-				gettingStartedGuide: 'https://blocknative.com',
-				explore: 'https://blocknative.com'
+				}
+				// gettingStartedGuide: 'https://blocknative.com',
+				// explore: 'https://blocknative.com'
```

Comment:
Gas net docs here?

Reply:
We could. I just don't like how the text labels read here plus I don't think it's that crucial to have these links display. Plenty of space to link to docs in the page where it's a lot more visible and clear.

Reply:
Sounds good, we can probably just remove this then